### DISCUSSION TITLE
Am I misunderstanding or is this the correct way of working?

### DISCUSSION THREAD

**Discussion by krabat1** [WordPress/gutenberg#51763] [Q&A] [answered]
![BCXSX](https://github.com/WordPress/gutenberg/assets/22174386/acb723ef-51c5-4e34-96a0-53110a9f43fa)
Version 6.2.2.
Theme: blankslate-based classic, not block-theme with bootstrap (but that's irrelevant because they wouldn't have implemented the Gutenberg editor if it only works with block themes)

I had make a 'row' in a post and i put it two image (horizontally and vertically centered), but it not work in live. In get the Inspector, and i see that html-comment with json-styles, but it not applied to .wp-block-group, it displayed as block element, instead of flex.
(That's not the only thing, of course)

Is this operation correct, or it is proof that something essential did not happen?
Should these comments appear in the source code?
If I eliminate functions.php and style.css in my theme, the problem persists.

What do i have to do to make the styles apply and these editorial comments disappear?
Debugging is turned on, I don't get any errors either from the server side or from the inspector.

Thank you for your help in advance!

**Comment by t-hamano** [comment]
Hi @krabat1,

As far as I know, the comment delimiter of the block should not be visible in the editor or on the front end.

I would like to confirm the following:

- What version of WordPress are you running, and do you have the Gutenberg plugin activated?
- Does this problem occur only in the editor or in the front end? Or both?

**Reply by krabat1** [reply]
Hi @t-hamano, thank you for taking the time to solve my problem.
I am using version 6.2.2, due to the above problems I did a fresh installation on a subdomain and uploaded the exported xml and theme to the new site.
I sadly found that the error persists when using my theme, but not with the Twenty Twenty-Three theme. In the official theme, everything works as it should, even without the Gutenberg plugin.
Furthermore, I see in the source (Twenty Twenty-Three) that new styles referring to Gutenberg appeared.
My conclusion is that my template is imperfect, maybe I missed something important. The theme is a child theme of blankslate (bootstrap-based), I've been developing a theme with it for about 10 years, but a lot has changed since then.
Even if I use the parent theme, all the reasons that confirm my hunch.
Unfortunately, since I don't get any error messages either from the server or from the console, I have nothing to hold on to in terms of moving forward, I can just start the development from the beginning and then I'll see where things fail.
Oh, using my theme, things works fine in the editor, but not in the frontend.

**Reply by t-hamano** [reply, marked as answer]
I have tried a bit of your theme in WordPress Playground and it appears to be working as expected.

https://playground.wordpress.net/?theme=blankslate

Perhaps some code in the child theme is affecting it.

![capture](https://github.com/WordPress/gutenberg/assets/54422211/84b89cbe-8369-4f34-bc74-2ae70966b544)

**Comment by krabat1** [comment]
Yeah, that's all I can think of too. Maybe I'll figure it out soon if I look at block templates in parallel and test the files of the child theme in order.
And of course I also learn something new, for example:
[https://fullsiteediting.com/lessons/adding-full-site-editing-features-to-classic-themes/](https://fullsiteediting.com/lessons/adding-full-site-editing-features-to-classic-themes/)
Let me thank you again for your help, my outlook is much better now!
